src/utils/index.ts

```diff
@@ -27,3 +27,5 @@ export function isTouch() {
       window.navigator.msMaxTouchPoints > 0,
   );
 }
+
+export * from "./types";
```

Comment:
Let's not reexport the types here again.. i would like to be explicit about importing the types from "./utils/types"

Reply:
Okay, that would be nice too. We know we are using the types. 👍